src/pages/studies/NewStudy.tsx

```diff
@@ -1,120 +1,86 @@
-};
+import { Container, Typography, Button, Box, Stepper, Step, StepLabel } from '@mui/material';
+import React from 'react';
+import { TagsDisplay } from '../../components/TagsDisplay';
+import { NewStudyJsonForm } from '../../components/NewStudyJsonForm';
+import { TagTrainingComponent } from '../../components/TagTrainingComponent';
+//import { AslLexSignBankField, aslLexSignBankControlRendererTester } from '../../custom-fields/asl-lex-field';
+//import { fileListControlRendererTester, FileListField } from '../../custom-fields/file-list';
+//import { VideoOptionUpload, videoOptionUploadRendererTester } from '../../custom-fields/video-option-upload.component';
+//import { userVideoOptionRendererTester, UserVideoOption } from '../../custom-fields/user-video-option-field.component';
+//import { VideoFieldComponent, videoFieldTester } from '../../video-recording/components/video-field.component';
+//import { TagFieldGeneratorService } from '../../services/tag-field-generator.service';
 
 const NewStudy: React.FC = () => {
-  const initialData = {
-    name: '',
-    description: '',
-    instructions: ''
+  //all constants
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
   };
 
-  const [data, setData] = useState(initialData);
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
 
-  const handleChange = (data: any) => {
-    setData(data);
+  const handleReset = () => {
+    setActiveStep(0);
   };
 
+  const steps = ['Study Identification', 'Construct Tagging Interface', 'Select Tag Training Items'];
+
+  function getSectionComponent() {
+    switch (activeStep) {
+      case 0:
+        return <NewStudyJsonForm />;
+      case 1:
+        return <TagsDisplay />;
+      case 2:
+        return <TagTrainingComponent />;
+      default:
+        return null;
+    }
+  }
+  //sx={{ '& .MuiContainer-root': { width: '550rem' } }}
   return (
-    <Container sx={{ left: '5%', width: '90%', top: '100px', position: 'absolute' }}>
-      <JsonForms schema={schema} uischema={uischema} data={data} renderers={materialRenderers} cells={materialCells} onChange={({ data }) => handleChange(data)} />
+    <Container sx={{ position: 'absolute', left: '-5%', right: '-5%', alignItems: 'baseline', width: '100%', top: '100px', display: 'flex', flexDirection: 'column' }}>
+      <Typography sx={{ margin: '10px 0px 15px 10px' }} variant="h5">
+        Create New Study
+      </Typography>
+      <Container sx={{ maxWidth: 'xl' }}>
+        <Stepper activeStep={activeStep}>
+          {steps.map((label) => {
+            const stepProps: { completed?: boolean } = {};
+            return (
+              <Step key={label} {...stepProps}>
+                <StepLabel>{label}</StepLabel>
+              </Step>
+            );
+          })}
+        </Stepper>
+        {activeStep === steps.length ? (
+          <React.Fragment>
```

Comment:
NIT: You don't have to change this, but in general we just use empty tags `<></>` to contain multiple children